### PR TITLE
feat: add custom cursor

### LINKED ISSUE #206: Add a gamified cursor pointer
Gamified cursor pointer like in https://imperiumgame.netlify.app/explore

### PR DESCRIPTION
**Pull request by ayushtom**
closes #206

### CODE REVIEW

File: styles/achievements.module.css [hunk 1]
```diff
@@ -80,11 +80,9 @@
 .backgroundFilter {
   position: absolute;
   border-radius: 30px;
-  background: linear-gradient(
-    90deg,
-    rgba(17, 16, 24, 0),
-    rgba(17, 16, 24, 0.51)
-  );
+  background: linear-gradient(90deg,
```
[the remark this answers is not in the document]
Comment (ayushtom) [reply styles/achievements.module.css]: yes. Can you dm me your config files? I'll replace them in my editor

**Review by irisdv — CHANGES_REQUESTED** [review]
Looks good ! Just, are you using the svg files for the cursor ?

**Comment by ayushtom** [comment]
removed extra svg files

**Review by irisdv — APPROVED** [review]
lgtm

**Pull request merged**